AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -2398,6 +2399,7 @@ function New-ADAccountForStorageAccount {
                         -Name $ADObjectName `
                         -AccountPassword $fileServiceAccountPwdSecureString `
                         -AllowReversiblePasswordEncryption $false `
+                        -PasswordNeverExpires $true `

```

Comment:
Is this wise? Users who have the ability to create a computer account might not have the ability to create a computer account with the password never expires flag set... I don't think I have the ability to create such a computer account in Redmond...

Reply:
I added -PasswordneverExpires to be consistent with what we have for New-ADUser. Does the same limitation apply to New-ADUser?

Reply:
I'm in agreement with Will.  I think we should keep it as-is (not marking password never expires for computers).  It's much more common to have non-expiring passwords for users and then to have forced password expiration on computers from what I've seen.  And we've tested that "expired" passwords on computer accounts won't stop the domain from issuing servicing tickets for the account encrypted with that expired password.

Reply:
Removed PasswordneverExpires